### PR TITLE
Shout at a user for mentioning whitep4nth3r.

### PR DESCRIPTION
**Pull request by SketchNI**

### CODE REVIEW

File: src/discord.ts [hunk 1]
```diff
@@ -95,6 +95,12 @@ discord.on(
 );
 
 discord.on("message", async (message: Message) => {
+  if(message.content.includes(`<@!${config.discord.whitep4nth3rId}>`)) {
+    if(message.author.id !== config.discord.p4nth3rb0tId) {
+      message.reply(`you are fined one credit for a violation of the Verbal Morality Statute.\n\nPlease refrain from mentioning whitep4nth3r directly if it is not extremely urgent. Thank you.`);
```
Comment (whitep4nth3r) [inline src/discord.ts:100]: Can we tag the person with maybe "Attention, @user!" and give the "you" a capital letter please?

Comment (whitep4nth3r) [reply src/discord.ts:100]: Oh but that won't work with the reply maybe. Disregard if nonsense.

Comment (SketchNI) [reply src/discord.ts:100]: The reply method prefixes with `@user,` which is why it starts that way. I could ditch the reply and create a new message with `Attention, @user! You are fined....` if you prefer it?

Comment (SketchNI) [reply src/discord.ts:100]: I could probably get rid of the meme text and display a proper warning since it is a pop culture reference?

Comment (whitep4nth3r) [reply src/discord.ts:100]: Yeah that might be better.

File: src/discord.ts [hunk 2]
```diff
@@ -97,7 +97,10 @@ discord.on(
 discord.on("message", async (message: Message) => {
   if(message.content.includes(`<@!${config.discord.whitep4nth3rId}>`)) {
     if(message.author.id !== config.discord.p4nth3rb0tId) {
-      message.reply(`you are fined one credit for a violation of the Verbal Morality Statute.\n\nPlease refrain from mentioning whitep4nth3r directly if it is not extremely urgent. Thank you.`);
+      message.channel.send(
+        `Attention <@!${message.author.id}>! Please refrain from mentioning ` +
+        `whitep4nth3r directly if it is not extremely urgent. Thank you.`
```
Comment (SketchNI) [inline src/discord.ts:102]: Split and concatenated the line to try and keep within the 120 character line limit I try to adhere to

**Pull request merged**